schemas/consensus_sequence.json

```diff
@@ -95,6 +114,16 @@
               "day"
             ]
           },
+          "sample_collection_date_null_reason": {
+            "enum": [
+              "Not Applicable",
+              "Missing",
+              "Not Collected",
+              "Not Provided",
+              "Restricted Access",
+              null
```

Comment:
@lindaxiang should null be here?

Reply:
if we have a field specifying that a user needs to include WHY they didn't include another field, should we allow them to submit nothing for that?

Reply:
@rosibaj 
- If `sample_collection_date` has input values, `sample_collect_data_null_reason` could be empty (null)
- If `sample_collection_date` is empty, user need to provide a reason in `sample_collect_data_null_reason`.
There is a conditional check after.

Reply:
see here: https://github.com/cancogen-virus-seq/metadata-schemas/blob/add_new_fields/schemas/consensus_sequence.json#L410-L435

Reply:
oh i see! thank you for explaining linda - i was reading this backwards!